conformance/setup.go

```diff
@@ -320,6 +323,9 @@ func init() {
 
 	skipEmptyLayerTest, _ = strconv.ParseBool(os.Getenv(envVarPushEmptyLayer))
 	deleteManifestBeforeBlobs, _ = strconv.ParseBool(os.Getenv(envVarDeleteManifestBeforeBlobs))
+	automaticCrossmountVal := ""
+	automaticCrossmountVal, runAutomaticCrossmountTest = os.LookupEnv(envVarAutomaticCrossmount)
+	automaticCrossmountEnabled, _ = strconv.ParseBool(automaticCrossmountVal)
```

Comment:
So if I understand correctly, the 201 test will only run if both `runAutomaticCrossmountTest` and `automaticCrossmountEnabled` are true?

and the 202 test if `runAutomaticCrossmountTest` is true and `automaticCrossmountEnabled` is false?

Reply:
Correct.